### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by ankitagithub0305** [community/community#53145] [GitHub Education]
My all informations are correct but u still not approved my developer pack 
Why ? Plz tell me what to do

**Comment by samus-aran** [comment]
Hi @ankitagithub0305  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
